Launcher/Storyboards/Base.lproj/Main.storyboard

```diff
@@ -1853,6 +1917,54 @@ Please choose one of the supported devices from "Test Runner" menu and click "St
             </objects>
             <point key="canvasLocation" x="1259" y="146"/>
         </scene>
+        <!--View Controller-->
+        <scene sceneID="ho4-Ml-Rym">
+            <objects>
+                <viewController id="Fzd-fW-W4C" sceneMemberID="viewController">
+                    <view key="view" id="JPz-gQ-5tw">
+                        <rect key="frame" x="0.0" y="0.0" width="193" height="44"/>
+                        <autoresizingMask key="autoresizingMask"/>
+                        <subviews>
+                            <textField verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="4WA-AF-Tik">
+                                <rect key="frame" x="8" y="-6" width="179" height="46"/>
+                                <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                                <textFieldCell key="cell" controlSize="mini" sendsActionOnEndEditing="YES" title="Identify directory to copy the build after downloading." id="UxM-nB-nnN">
```

Comment:
~~Would it make sense to call it: Location of the application under test. ? Not everyone might download the app like we do~~